src/main/java/com/bloomtechlabs/fp/entities/EducationHistory.java

```diff
@@ -0,0 +1,88 @@
+package com.bloomtechlabs.fp.entities;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Table(name ="education_histories")
+public class EducationHistory {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "client_id")
+    private Long clientId;
+
+    @Column(name = "school_name")
+    private String schoolName;
+
+    @Column(name = "level")
+    private String level;
+
+    @Column(name = "start_date")
+    @Temporal(TemporalType.TIMESTAMP)
+    private Date startDate;
+
+    @Column(name = "end_date")
+    @Temporal(TemporalType.TIMESTAMP)
+    private Date endDate;
+
+    public EducationHistory() {
+    }
+
+    public EducationHistory(Long clientId, String schoolName, String level, Date startDate, Date endDate) {
+        this.clientId   = clientId;
+        this.schoolName = schoolName;
+        this.level      = level;
+        this.startDate  = startDate;
+        this.endDate    = endDate;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
```

Comment:
This setter can change the primary key auto generated by JPA.

Reply:
looks like I had an oversite generating getters and setters. I'll fix this.